content/en/docs/setup/release/notes.md

```diff
@@ -250,6 +250,7 @@ The main themes of this release are:
 - Introduction of the pod overhead feature to the scheduler. This functionality is alpha-level as of
   Kubernetes v1.16, and is only honored by servers that enable the PodOverhead feature.gate. ([#78319](https://github.com/kubernetes/kubernetes/pull/78319), [@egernst](https://github.com/egernst))
 - Ephemeral containers have been added in alpha. These temporary containers can be added to running pods for purposes such as debugging, similar to how `kubectl exec` runs a process in an existing container. Also like `kubectl exec`, no resources are reserved for ephemeral containers and they are not restarted when they exit. Note that container namespace targeting is not yet implemented, so [process namespace sharing](https://kubernetes.io/docs/tasks/configure-pod-container/share-process-namespace/) must be enabled to view process from other containers in the pod. ([#59484](https://github.com/kubernetes/kubernetes/pull/59484), [@verb](https://github.com/verb))
+- EvenPodsSpread feature has been added in alpha. This feature introduced an API field `pod.spec.topologySpreadConstraints` to enable users control how Pods are spread across the cluster among failure-domains. ([#77327](https://github.com/kubernetes/kubernetes/pull/77327), [#77760](https://github.com/kubernetes/kubernetes/pull/77760), [#77828](https://github.com/kubernetes/kubernetes/pull/77828), [#79062](https://github.com/kubernetes/kubernetes/pull/79062), [#80011](https://github.com/kubernetes/kubernetes/pull/80011), [#81068](https://github.com/kubernetes/kubernetes/pull/81068), [@Huang-Wei](https://github.com/Huang-Wei))
```

Comment:
Does this change need to go into https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.16.md first?

Also, here's a suggested rewording:
```markdown
- Pod spread constraints have been added in alpha. You can use these constraints to control how Pods are spread across the cluster among failure-domains. ([#77327](https://github.com/kubernetes/kubernetes/pull/77327), [#77760](https://github.com/kubernetes/kubernetes/pull/77760), [#77828](https://github.com/kubernetes/kubernetes/pull/77828), [#79062](https://github.com/kubernetes/kubernetes/pull/79062), [#80011](https://github.com/kubernetes/kubernetes/pull/80011), [#81068](https://github.com/kubernetes/kubernetes/pull/81068), [@Huang-Wei](https://github.com/Huang-Wei))
```

Reply:
Thanks for the hint, I have a parallel PR open in k/k and the release notes website:
- https://github.com/kubernetes/kubernetes/pull/82868
- https://github.com/kubernetes-sigs/release-notes/pull/103

Regarding the rewording: We can do that, but I would have to change 3 PRs, (where one is not mine) to keep it in sync. Should I go for this effort or do you think we can leave it as it is?

Reply:
I prefer release notes that describe the feature itself rather than naming the related feature gate, but it's not a huge deal.

Reply:
Changed it on all three PRs, except the one which I do not own: https://github.com/kubernetes/sig-release/pull/801

Reply:
@saschagrunert I've updated the k/sig-release one to keep the same words

BTW: I opened a PR in k/k (https://github.com/kubernetes/kubernetes/pull/82853) before yours :) I'm now chasing a root owner to approve that. 🤞